docs/pages/guides/meshery-metrics.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: guide
+title: meshery metrics
+description: How to use meshery metrics
+permalink: guides/mesherymetrics
+type: guide
+---
+
+Inorder to generate reports/load testing of your k8 applications, meshery uses `Grafana` and `Prometheus`.
+Here you can find steps to generate these endpoints in different environemnts/service mesh adaptors.
+
+* TOC
+{:toc}
+
+
+### Istio + Minikube(dev env)
+
+
+
+In order for Meshery to gain access to grafana and prometheus, their network ports need to be exposed.
+
+
+
+**1) Get minikube ip**
+
+Get Minikube ip by executed below command.
+    
+```        
+    $minikube ip
+    o/p: 172.17.0.2
+```
+<i>Note: Here Istio is installed in `istio-system` name space and `productpage` sample app is installed in `book-info` name space.
+</i>
+
+**2)Expose Prometheus service**
+
+a)By default  `prometheus`  spec type is configured to `ClusterIP`  you can chage it to `NodePort` using below command.
```

Comment:
```suggestion
a)By default  `prometheus`  spec type is configured to `ClusterIP`  you can change it to `NodePort` using below command.
```

Reply:
Also, can we use an unordered list here rather than a, b, c sub listings?

Reply:
yep, we can try that in the next version.